index.html

```diff
@@ -93,3 +93,424 @@ <h2 class="subSentenceText">I'm a Software Developer</h2>
+                <img src="Rectangle.png" alt="Rectangle" /> Language
+              </li>
+              <ul class="TechStacks techstacks1D">
+                <li>JavaScript</li>
+                <li>Ruby</li>
+                <li>HTML</li>
+                <li>CSS</li>
+              </ul>
+            </ul>
+
+            <ul class="tech2">
+              <li class="Language1">
+                <img src="Rectangle2.png" alt="Rectangle" /> Frameworks
+              </li>
+              <ul class="TechStacks">
+                <li>React.js</li>
+                <li>Ruby on Rails</li>
+                <li>RSpec</li>
+                <li>Capybara</li>
+              </ul>
+            </ul>
+
+            <ul class="tech">
+              <li class="Language1">
+                <img src="Rectangle3.png" alt="Rectangle" /> skills
+              </li>
+              <ul class="TechStacks">
+                <li>Database Management</li>
+                <li>Version Control</li>
+                <li>CLI</li>
+                <li>API Design</li>
+                <li>Web Developement</li>
+              </ul>
+            </ul>
+          </ul>
+        </div>
+      </div>
+    </section>
+
+    <!-- html form -->
+
+    <section class="Contact" id="Contact">
+      <div class="form">
+        <a href="#form"></a>
+        <div class="mainContent">
+          <h2 class="smallPara">
+            I'm always interested in hearing about new projects, so if you'd
+            like to chat please get in touch.
+          </h2>
+          <form
+            class="firstForm"
+            id="form"
+            method="post"
+            action="https://formspree.io/f/xgedqoye"
+          >
+
+            <ul class="input">
+              <div class="subInput">
+              <li class="newInputName">
+                <input
+                  name="name"
+                  type="text"
+                  id="Name"
+                  placeholder="Full name"
+                  required
+                />
+                <div class="gaps1"></div>
+              </li>
+
+              <li class="newInputEmail">
+                <input
+                  name="Email"
+                  type="Email"
+                  id="Email"
+                  placeholder="Email address"
+                  required
+                />
+                <div class="gaps2"></div>
+              </li>
+
+              <li class="textArea" id="message">
+                <textarea
+                  class="textArea1"
+                  name="message"
+                  id="textArea1"
+                  placeholder="Write me something..."
+                  maxlength="500"
+                  required
+                ></textarea>
+              </li>
+
+              <li>
+                <button class="submit" type="submit" value="submit">
+                  Get In Touch
+                </button>
+              </li>
+            </div>
+            </ul>
```

Comment:
- [ ] Great job on incorporating flexbox for the input elements! However, as mentioned in the previous review, the gap between the input elements needs to be adjusted to match the design specifications. Currently, the margin is set to `16px`, but it should be `24px`. To achieve the desired layout, I recommend using flexbox along with the appropriate gap properties or margins to create the desired spacing between the input elements. Keep up the great work! 💪🌟

![image](https://github.com/bhushan354/My-Portfolio/assets/55840999/426afce0-d054-4441-a821-a84f35a865ed)